english/light-fire/fire.md

```diff
@@ -10,7 +10,7 @@ This situation has the potential to go terribly wrong.
 
 The walls around you begin to melt, globs of flaming marshmallow falling all around you. What do you do?
 
-As you somehow avoid the globs of marshmallow using your reflexes, you notice a big enough opening in the melted 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.